composer.json

```diff
@@ -44,16 +44,16 @@
         "ext-curl": "*",
         "amphp/amp": "^2.4.2",
         "bamarni/composer-bin-plugin": "^1.2",
-        "brianium/paratest": "^4.0.0",
+        "brianium/paratest": "^4.0||^6.0",
         "phpdocumentor/reflection-docblock": "^5",
-        "phpmyadmin/sql-parser": "5.1.0",
+        "phpmyadmin/sql-parser": "5.1.0||dev-master",
```

Comment:
It might be easier to remove that single test that uses this parser, and drop the dependency. As of now, `sql-parser` has no stable PHP 8 version.